templates/cron.sol11.erb

```diff
@@ -0,0 +1,8 @@
+# This file is being maintained by Puppet.
+# DO NOT EDIT
+account definitive        pam_user_policy.so.1
+<% if @ensure_vas == 'present' -%>
```

Comment:
why are there variable here in the template when there is already conditional logic in the manifest to set the contents of this file?

These should look like the other templates.